com.alpineclient.AlpineClient.yml

```diff
@@ -0,0 +1,33 @@
+id: com.alpineclient.AlpineClient
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: alpine-client
+finish-args:
+  - "--persist=.alpineclient"
+  - "--socket=x11"
+  - "--socket=pulseaudio"
+  - "--device=dri"
+  - "--share=network"
+  - "--share=ipc"
+  - "--filesystem=xdg-download:ro" # Drag & drop mods
+  - "--filesystem=xdg-run/app/com.spotify.Client:create" # Spotify integration
+  - "--filesystem=xdg-run/app/com.discordapp.Discord:create" # Discord integration
+modules:
+  - name: xrandr
+    sources:
+      - type: archive
+        url: "https://xorg.freedesktop.org/archive/individual/app/xrandr-1.5.1.tar.xz"
+        sha256: "7bc76daf9d72f8aff885efad04ce06b90488a1a169d118dea8a2b661832e8762"
+
+  - name: alpineclient
+    buildsystem: simple
+    build-commands:
+      - "install -D -m 755 alpine-client /app/bin/alpine-client"
```

Comment:
this file isn't found. maybe you actually forgot to build it?

Reply:
I'm sorry, I somehow forgot to commit the rest of my changes. Should be fixed now.